packages/compiler/templates/emitter-ts/src/emitter.ts

```diff
@@ -0,0 +1,10 @@
+import { EmitContext, emitFile, resolvePath } from "@typespec/compiler";
+
+export async function $onEmit(context: EmitContext) {
+  if (!context.program.compilerOptions.noEmit) {
```

Comment:
Might it be worth putting something super simple here with the emitter framework, like just printing out model names or some such?

Reply:
I wasn't sure I really wanted people to try to use the emitter framework today as there is still a lot of issues with it.

I might take a stab during the holidays at solving all those issues Travis and I got when using it and after I think we could setup the boilperplate for the emitter framework usage.